src/items/LinkItem.tsx

```diff
@@ -105,6 +95,10 @@ const LinkItem: FC<LinkItemProps> = ({
     // }
   };
 
+  const onClick = (): void => {
+    redirect(url, { openInNewTab: true });
+  };
+
   const component = (
     <React.Fragment>
```

Comment:
I would import { Fragment } from React. Looks nicer and we're already importing things.